pluto_build/bin/config.sh

```diff
@@ -29,12 +29,14 @@ function urlparse {
     BUILD_PORT="$(echo $hostport | sed -e 's,^.*:,:,g' -e 's,.*:\([0-9]*\).*,\1,g' -e 's,[^0-9],,g')"
     BUILD_DB="$(echo $url | grep / | cut -d/ -f2-)"
 }
+urlparse $BUILD_ENGINE
```

Comment:
why have urlparse outside of the export functions? Isn't the role of this function call to set arguments for the docker run?

Reply:
good catch this was me trying to figure out whether urlparse was working. Deleting now.